the-blue-alliance-ios/View Controllers/Events/EventRankingsTableViewController.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  EventRankingsTableViewController.swift
+//  the-blue-alliance-ios
+//
+//  Created by Zach Orr on 5/14/17.
+//  Copyright © 2017 The Blue Alliance. All rights reserved.
+//
+
+import Foundation
+import TBAKit
+import UIKit
+import CoreData
+
+class EventRankingsTableViewController: TBATableViewController {
+
+    override var persistentContainer: NSPersistentContainer! {
+        didSet {
+            updateDataSource()
+        }
+    }
+    var event: Event!
+    var rankingSelected: ((EventRanking) -> ())?
+    
+    // MARK: - View Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.register(UINib(nibName: String(describing: EventRankingTableViewCell.self), bundle: nil), forCellReuseIdentifier: EventRankingTableViewCell.reuseIdentifier)
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        clearsSelectionOnViewWillAppear = splitViewController!.isCollapsed
+    }
+    
+    // MARK: - Refreshing
+    
+    override func refresh() {
+        removeNoDataView()
+        
+        // First things first... refresh all teams for the district, *then* fetch their rankings
```

Comment:
Not that this is a *bad* way to go about fetching relevant data that we need, I've just been rethinking how we actually want to do this